main.js

```diff
@@ -5,24 +5,25 @@ function update_releases()
     fetch('https://api.github.com/repos/skiftOS/skift/releases')
```

Comment:
Can you do a pull request with only these changes

Reply:
yes :)